conf/zoo_sample.cfg

```diff
@@ -6,6 +6,10 @@ initLimit=10
 # The number of ticks that can pass between 
 # sending a request and getting an acknowledgement
 syncLimit=5
+# enable TTL Nodes
+# IMPORTANT: when enabled, your server ID cannot be greater than 254
```

Comment:
Same comment I left on the JIRA ticket: shouldn't it be 127, not 254? My understanding of the problem was that the whole high bit was being used, not the whole byte sequence of 255.

Reply:
@DanBenediktson 127 is the case with the existing bug. I re-wrote how this is handled so that now it's 255. The high bit is now used to signal that the ephemeralOwner is special.